scripts/ec2cleanup.py

```diff
@@ -129,7 +132,34 @@ def delete_unused_network_interfaces(provider_mgmt, excluded_enis, output):
         logger.exception('Exception in %r', delete_unused_network_interfaces.__name__)
 
 
-def ec2cleanup(exclude_volumes, exclude_eips, exclude_elbs, exclude_enis, output):
+def delete_stacks(provider_mgmt, excluded_stacks, output):
+    stack_list = []
+    provider_name = provider_mgmt.kwargs['name']
+    try:
+        for stack in provider_mgmt.list_stacks():
+            if excluded_stacks and stack.stack_name in excluded_stacks:
+                logger.info("  Excluding Stack name: %r", stack.stack_name)
+                continue
+            else:
+                today = datetime.utcnow().replace(tzinfo=None)
+                some_date = today - timedelta(days=1)
+                if stack.creation_time < some_date:
+                    stack_list.append([provider_name, stack.stack_name])
+                    provider_mgmt.delete_stack(stack.stack_name)
+                    # TODO: Remove once we have reliable Trottling-resistant aproach
+                    sleep(5)
+        logger.info("  Deleted CloudFormation Stacks: %r", stack_list)
+        with open(output, 'a+') as report:
+            if stack_list:
+                report.write(tabulate(tabular_data=stack_list,
```

Comment:
Q: are you sure this report is necessary at all ? I'd rather use logger instead.

Reply:
Purpose of this PR add stack removal. I guess, we might add GH issue to rework this module in the future to use logger